### ISSUE TITLE
Please add a LICENSE

### ISSUE THREAD

**Issue by JJ** [cristgl/TFG#1]
At it should be compatible with the upstream code you're using.

**Comment by cristgl** [comment]
I think I have added the correct license.

**Comment by JJ** [comment]
Right, that's correct: https://opensource.stackexchange.com/a/5471/8500 Please check that the rest of the source does not have a license that's not compatible with the GPL.

**Comment by cristgl** [comment]
I have been checking it and I think it is compatible with all the source.
